cage.1.scd

```diff
@@ -0,0 +1,81 @@
+cage(1)
+
+# NAME
+
+cage - a Wayland kiosk compsitor
+
+# SYNOPSIS
+
+*cage* [-dhrv] [--] _application_ [application argument ...]
+
+# DESCRIPTION
+
+Cage runs a single, maximized application. User interaction and activities
+outside the scope of the running application are prevented.
+
+As such, user input such as moving, resizing, minimizing and unmaximizing
```

Comment:
I'm not sure if this description belongs in a man page; what do you think? I think perhaps the first two lines here are enough.

If the additional description does belong here, could you take the description from the Wiki? The website is slightly outdated and I intend on updating that to the description in the Wiki as well.

Reply:
It's the same on the website and wiki (second paragraph on the main page). Also do you mean remove the first sentence or the whole paragraph? For now, I did the latter.

Reply:
removed everything except for the first two sentences. not sure if that's what you wanted, but you never clarified.

Reply:
Sorry for not replying to you sooner @travankor. I don't think it matters that much at this point; the first line isn't exactly true anymore either. Cage can now run multiple applications, but only a single one is visible at any point in time. I'll accept the PR and then I'll update the description on the website, wiki and man page once I get closer to a new release.